Sources/SemanticIndex/UpdateIndexStoreTaskDescription.swift

```diff
@@ -264,72 +276,150 @@ package struct UpdateIndexStoreTaskDescription: IndexTaskDescription {
       )
     }
 
+    // Compute the partitions within which we can perform multi-file indexing. For this we gather all files tha may only
+    // be indexed by itself in `partitions` an collect all other files in `fileInfosByBuildSettings` so that we can
+    // create a partition for all files that share the same build settings.
```

Comment:
"For this we gather all the files that may only be indexed by themselves...". Also no need to fix the comment nit picks in this PR.